akka-stream-tests-tck/src/test/scala/akka/stream/tck/Timeouts.scala

```diff
@@ -10,6 +10,6 @@ object Timeouts {
 
   def publisherShutdownTimeoutMillis: Int = 3000
 
-  def defaultTimeoutMillis: Int = 800
+  def defaultTimeoutMillis: Int = 1600
```

Comment:
This change can add I think 10-20 minutes to build-times, so this is a no-go until the fix linked by @patriknw is fixed.